terraform/prometheus.tf.sample

```diff
@@ -9,13 +13,15 @@ module "prometheus" {
   env                      = "($ environment $)"
   vpc_id                   = "<vpc-id>"
   bastion_sg_allow         = "<bastion-sg>"
-  rds_password             = "${var.rds_password}"
+  # rds_password             = "${var.rds_password}"
```

Comment:
Maybe add a quick comment to say to uncomment rds_password variable and output + this line to use rds and set the password.